kin-core/src/main/java/kin/core/KinAccount.java

```diff
@@ -104,6 +106,63 @@ TransactionId sendTransactionSync(@NonNull String publicAddress, @NonNull BigDec
     @NonNull
     Balance getBalanceSync() throws OperationFailedException;
 
+    /**
+     * Create {@link Request} for getting all the transactions payments history for the current account.
+     * <p> See {@link KinAccount#getPaymentsHistorySync()} for possibles errors</p>
+     *
+     * @return {@code Request<List<PaymentInfo>> } PaymentInfo - the payment information for a specific payment operation in a transaction
+     */
+    @NonNull
+    Request<List<PaymentInfo>> getPaymentsHistory();
+
+    /**
+     * For method description see {@link KinAccount#getPaymentsHistory()}
+     * @param requestParams an optional parameters to the request, such as accountId, cursor, limit, and order.
+     * Create {@link Request} for getting all the transactions payments history for a given account.
+     *                        If no account has been given to the requestParams then it will be for the current account.
+     * <p> See {@link KinAccount#getPaymentsHistorySync()} for possibles errors</p>
+     *
+     * <p><b>Note:</b>     requestParams can have the next members:</p>
+     * <p><b>accountId</b> is optional, a string, if not given then using the current account if exist. It is represents the ID of an account.</p>
+     * <p><b>cursor:</b>   is optional, a paging token, specifying where to start returning records from. for example 12884905984.</p>
```

Comment:
I'm not sure how token can be used here? what actually it is?
We should give the option to get this token from every payment o.w. it is not useful currently